crates/bevy_sprite/src/collide_aabb.rs

```diff
@@ -16,6 +16,11 @@ pub enum Collision {
 /// * `a_pos` and `b_pos` are the center positions of the rectangles, typically obtained by
 /// extracting the `translation` field from a `Transform` component
 /// * `a_size` and `b_size` are the dimensions (width and height) of the rectangles.
+///
+/// The collision side is determined based on the position of the second rectangle "B":
+/// * `Left` means left of "B", `Top` means top of "B", ...
+/// * If the collision appears on multiple sides then the side which stretches the farthest is returned
+/// unless all sides are involved (`Inside`)
```

Comment:
```suggestion
/// The return value is the side of `B` that `A` has collided with. `Left` means that
/// `A` collided with `B`'s left side. `Top` means that `A` collided with `B`'s top side.
/// If the collision occurs on multiple sides, the side with the deepest penetration is returned.
/// If all sides are involved, `Inside` is returned.
```
The new language in the `collide` docs actually confused me a fair bit. I really wasn't sure if the "side" belonged to rect A or B, and it wasn't obvious what "stretched" meant.

Something like this is making a bit more sense to me.

Reply:
I like the suggestion. Thank you!